en/docs/deploy-with-wso2-choreo.md

```diff
@@ -0,0 +1,8 @@
+# Deploy with WSO2 Choreo
+
+You can manage your Ballerina projects by deploying them in [WSO2 Choreo](https://wso2.com/choreo/). Integrating the [Choreo VS Code extension](https://marketplace.visualstudio.com/items?itemName=WSO2.choreo) with the Ballerina VS Code extension enables this functionality. 
```

Comment:
Is it fine to say **integrate** choreo vs code extension to Ballerina vs code extension?

Reply:
Changed to `The combination of the Ballerina VS Code extension and [Choreo VS Code extension](https://marketplace.visualstudio.com/items?itemName=WSO2.choreo) enables this functionality. `